docker/01_bootstrap.sh

```diff
@@ -55,32 +49,4 @@ PGPASSWORD=$SUPERUSER_PASSWORD psql -U postgres -d template1 -c "CREATE EXTENSIO
 PGPASSWORD=$SUPERUSER_PASSWORD psql -U postgres -d template1 -c "CREATE EXTENSION IF NOT EXISTS svector CASCADE;"
 PGPASSWORD=$SUPERUSER_PASSWORD psql -U postgres -d template1 -c "CREATE EXTENSION IF NOT EXISTS vector CASCADE;"
 
-# We collect basic, anonymous telemetry to help us understand how many people are using
-# the project. We only do this if PARADEDB_TELEMETRY is set to "true"
-if [[ "$PARADEDB_TELEMETRY" == "true" ]] && [ "$POSTHOG_API_KEY" != "" ] && [ "$POSTHOG_HOST" != "" ]; then
```

Comment:
Is it possible to keep this? I would like for the "ParadeDB Deployment" to still get sent. This allows us to easily track number of new deployments, which is a useful metric

Reply:
This is still happening, it's just done by the background worker now in pgrx/Rust.

Reply:
Or, if that's too complicated by requiring yet an extra env var, we can remove this... But it would be good to have a way to know whether a deployment is the extension standalone vs the Dockerfile, because that gives us very useful information